core/vm/actor/builtin/states/market_actor_state.hpp

```diff
@@ -37,9 +37,9 @@ namespace fc::vm::actor::builtin::states {
 
     using DealSet = adt::Set<UvarintKeyer>;
 
-    adt::Array<DealProposal> proposals;
-    adt::Array<DealState> states;
-    adt::Map<DealProposal, CidKeyer> pending_proposals;
+    adt::Array<DealProposal, 5> proposals;
+    adt::Array<DealState, 6> states;
+    adt::Set<CidKeyer> pending_proposals;
```

Comment:
Why adt::Set ?

Reply:
Because specs-actors say it's `Map` to `null`

Reply:
FIXME: v0 was not `Set`.
TODO: document types used in specs-actors versions